src/webgl/p5.Shader.js

```diff
@@ -316,9 +316,10 @@ p5.Shader = class {
   }
 
   _setMatrixUniforms() {
+    const modelMatrix = this._renderer.uModelMatrix;
     const viewMatrix = this._renderer._curCamera.cameraMatrix;
```

Comment:
Should be the same behaviour for now, but for consistency, can we read the new view matrix instead of directly from the camera?
```suggestion
    const viewMatrix = this._renderer.uViewMatrix;
```

Reply:
Similarly, below, where it currently has
```js
    if (this.uniforms.uCameraRotation) {
      this._renderer.curMatrix.inverseTranspose(this._renderer.
_curCamera.cameraMatrix);
```
Can we change it to:
```js
    if (this.uniforms.uCameraRotation) {
      this._renderer.curMatrix.inverseTranspose(this._renderer.uViewMatrix);
```